clang/include/clang/Lex/HeaderSearchOptions.h

```diff
@@ -219,6 +219,12 @@ class HeaderSearchOptions {
 
   unsigned ModulesValidateDiagnosticOptions : 1;
 
+  /// Whether to entirely skip writing diagnostic options.
+  unsigned ModulesSkipDiagnosticOptions : 1;
+
+  /// Whether to entirely skip writing header search paths.
```

Comment:
Then let's try to add comments about these two options are primarily used for deps scanning. It is helpful for future developers.